osu.Game/Overlays/Dashboard/CurrentlyOnlineDisplay.cs

```diff
@@ -120,7 +118,7 @@ protected override void OnFocus(FocusEvent e)
             searchTextBox.TakeFocus();
         }
 
-        private void onUserPresenceUpdated(object sender, NotifyDictionaryChangedEventArgs<int, UserPresence> e) => Schedule(() =>
+        private void onUserPresenceUpdated(object? sender, NotifyDictionaryChangedEventArgs<int, UserPresence> e) => Schedule(() =>
```

Comment:
With the recent changes to split out local user state, this is now missing self (you can no longer see your own panel on the online display). This can be fixed by decoupling the local list and inserting self, but I'm thinking maybe it makes more sense to keep self in the all-presences list? Interested in opinions on this one @ppy/team-client.

Reply:
I have a vague unsubstantiated recollection that when I worked on the user presence code I initially excluded self from the online display list, and then you added it back, but I'm not sure if that even really happened, I couldn't find any evidence of it on a few searches.

I'm relatively ambivalent on showing own panel. I don't think showing self brings any value, but also doesn't really seem to have any real downside, so I'm not sure I have a formed opinion at the end of the day.

Reply:
Not to my knowledge. In this particular conversation I was attempting to chip in on whether being able to see self on this display was relevant or not.

The backlink here from my PR was https://github.com/ppy/osu-server-spectator/pull/259#issuecomment-2597771812, which has nothing to do with this particular review thread (but in general is relevant to this PR, namely the concern in https://github.com/ppy/osu/pull/31524#issuecomment-2597727102).

The lack of any response to either point was why I didn't attempt picking up reviewing this any further because these look like valid concerns to me.

Reply:
Never mind, both issues are still present.